drivers/dma/dma_sedi.c

```diff
@@ -350,7 +350,7 @@ static int dma_sedi_stop(const struct device *dev, uint32_t channel)
 	return 0;
 }
 
-static const struct dma_driver_api dma_funcs = { .config = dma_sedi_chan_config,
+static DEVICE_API(dma, dma_funcs) = { .config = dma_sedi_chan_config,
 						 .start = dma_sedi_start,
 						 .stop = dma_sedi_stop,
 						 .reload = dma_sedi_reload,
```

Comment:
Since we're touching this, add a trailing comma to the last field and indent all fields with a single tab.